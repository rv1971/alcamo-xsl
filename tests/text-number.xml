<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <xsl:stylesheet
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xml:id="xslt"
      version="1.0">
    <xsl:import href="../xsl/text.xsl"/>

    <xsl:template match="/">
      <a:result>
        <xsl:apply-templates select="/a:doc/a:data"/>
     </a:result>
    </xsl:template>

    <xsl:template match="a:item">
      <a:item>
        <xsl:value-of select="."/>
        <xsl:value-of select="' '"/>
        <xsl:apply-templates select="." mode="a:number"/>
      </a:item>
    </xsl:template>
  </xsl:stylesheet>

  <a:data>
    <a:item>1</a:item>
    <a:item>003</a:item>
    <a:item>9</a:item>
    <a:item>10</a:item>
    <a:item>11</a:item>
    <a:item>20</a:item>
  </a:data>
</a:doc>
