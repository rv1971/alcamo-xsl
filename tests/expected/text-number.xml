<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <a:item>1 one</a:item>
  <a:item>3 three</a:item>
  <a:item>9 nine</a:item>
  <a:item>10 ten</a:item>
  <a:item>11 11</a:item>
  <a:item>20 20</a:item>
</a:result>
